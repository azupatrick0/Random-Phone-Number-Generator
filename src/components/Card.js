import React from 'react';

const Card = ({ title, cardValue}) => (
  <div style={{
    height: '25vh',
    borderRadius: 4,
    width: '20vw',
    boxShadow: '1px 1px 4px plum',
    backgroundColor: 'white',
    textAlign: 'center',
    paddingTop: '1vh',
    fontWeight: 'bold',
  }}>
    <div style={{ fontSize: 20 }}>{title}</div>
    <div style={{paddingTop: '4vh', fontSize: 40, fontWeight: 'bold'}}>{cardValue}</div>
  </div>
)

export default Card;
