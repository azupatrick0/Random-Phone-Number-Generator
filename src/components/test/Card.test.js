import React from 'react';
import Card from '../Card';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallow } from 'enzyme';

configure({ adapter: new Adapter() });

it('renders Card component without crashing', () => {
  const wrapper = shallow(<Card />);
  expect(wrapper).toMatchSnapshot();
});
