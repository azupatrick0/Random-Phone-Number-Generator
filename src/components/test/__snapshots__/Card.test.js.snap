// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Card component without crashing 1`] = `ShallowWrapper {}`;
